docs/management/index-lifecycle-policies/example-index-lifecycle-policy.asciidoc

```diff
@@ -1,23 +1,159 @@
+** Set replicas to 1
+** Force merge indices to a single segment
+* Delete logs after 90 days
+
+
+[float]
+==== Prerequisites
+
+To complete this tutorial, you'll need:
+
+* An {es} cluster with hot and warm nodes configured for shard allocation
+awareness. If you’re using https://www.elastic.co/guide/en/cloud/current/ec-getting-started-templates-hot-warm.html[Elasticsearch Service],
+choose the hot-warm architecture deployment template.
+
++
+For a self-managed cluster, use https://www.elastic.co/guide/en/elasticsearch/reference/current/allocation-awareness.html[shard allocation awareness]
+to label data nodes as hot or warm. This step is required to migrate shards between
+nodes using shard allocation awareness.
++
+For example, you can set this in `elasticsearch.yml` for each data node:
++
+[source,yaml]
+--------------------------------------------------------------------------------
+node.attr.data: "warm"
+--------------------------------------------------------------------------------
+
+* A server with Filebeat installed and configured to send logs to the `elasticsearch`
+output as described in https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-getting-started.html[Getting Started with Filebeat]
+
+[float]
+==== View the filebeat ILM policy
+
+Filebeat includes a default ILM policy which enables rollover. Index lifecycle
+management is enabled automatically if you’re using the default `filebeat.yml`
+and index template.
+
+To view the default policy in {kib}, go to *Management > Index Lifecycle Policies*,
+search for _filebeat_, and choose the _filebeat-version_ policy.
+
+This policy initiates the rollover action when the index size reaches 50G or 30
+days old.
+
+[role="screenshot"]
+image::images/tutorial-ilm-hotphaserollover-default.png["Default policy"]
+
+
+[float]
+==== Modify the policy
+
+The default policy is enough to prevent the creation of many tiny daily indices.
+You can modify the policy to meet more complex requirements.
+
+* Activate the warm phase.
+
++
+You have a choice to make, you can’t meet the requirements exactly with rollover
+enabled in the hot phase.
+
+
+** Enable *move to warm phase on rollover*. The index may move to the warm phase
+more quickly than intended if it reaches the *Maximum index size* condition before
+*Maximum age*.
+
+** Provide a value for *Timing for warm phase*. Setting this to *15* will keep the
+indices on hot nodes for a range of  15-45 days, depending on when the initial
+rollover occurred.
+
+* *Select a node attribute to control shard allocation*. For this example,
+choose `data:warm(2)` to migrate shards to warm data nodes.
+
+*  Change *Number of replicas* to *1*.
++
+NOTE:  When rollover is enabled in the hot phase, action timing in the other phases
+is based on the rollover date.
+
++
+[role="screenshot"]
+image::images/tutorial-ilm-modify-default-warm-phase-rollover.png["Modify to add warm phase"]
+
+* Activate the delete phase.
```

Comment:
Make this one step.  

Activate the delete phase and set the **Timing** to **60** days.